test_prebuild/test_unit_conv/ccpp_prebuild_config.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+# CCPP prebuild config for GFDL Finite-Volume Cubed-Sphere Model (FV3)
+
+###############################################################################
+# Definitions                                                                 #
+###############################################################################
+
+HOST_MODEL_IDENTIFIER = "FV3"
```

Comment:
It's a little weird that the host model identifier is FV3 since this is a unit test that arguably has nothing to do with FV3.

Reply:
I know, it's always been like that. Will go away when prebuild dies ....